### ISSUE TITLE
[Postgres Dialect] Late Binding Views Treated as Alias

### ISSUE THREAD

**Issue by will-misslin** [sqlfluff/sqlfluff#717]
## Expected Behaviour

Using Postgres, a late-binding view includes a `with` statement at the conclusion of the 

```sql
create my_schema.my_view as
select * from schema.table
with no schema binding;
```

## Observed Behaviour

Using `sqlfluff fix` on this triggers L003 (missing indent) and L011 (missing explicit aliasing)

```sql
create my_schema.my_view as
select * from schema.table
	as	with no schema binding;
```

This isn't valid SQL and this line should not be indented.

## Version
Include the output of `sqlfluff --version` along with your Python version

`sqlfluff, version 0.3.6`

## Configuration
```
[sqlfluff]
dialect = postgres
exclude_rules = L007,L027,L029

[sqlfluff:rules]
max_line_length = 79
indent_unit = tab

[sqlfluff:templater:jinja:context]
year=2020
month=01
day=01
schema=my_schema
table=my_table
object=my_object

[sqlfluff:rules:L010]
capitalisation_policy = lower
```
